data/operators/amenity/charging_station.json

```diff
@@ -788,6 +788,14 @@
         "operator": "Daimler AG"
       }
     },
+    {
+      "displayName": "da emobil GmbH & Co KG",
```

Comment:
There should be a capital letter of firast letter of words so "Da Emobil ..."

Reply:
Here the display name should just be "da emobil".  This is the name that you'd see in the JOSM menu, or in the iD preset, not the legal name.